### DISCUSSION TITLE
how to run gptp on zephyr environment

### DISCUSSION THREAD

**Discussion by aravind-gara** [zephyrproject-rtos/zephyr#39680] [Q&A] [answered]
Hi, we want to use gPTP stack with NXP FRDM K64F.
But we need to know if we can use this in production or is this stack in early development stage ?

**Comment by jukkar** [comment, marked as answer]
Answered this also on discord. The summary is that the gPTP stack is not formally tested or certified. It has been used successfully with frdm-k64f and sam-e70, and things seem to work as expected. For use in a product, you need to verify / analyze yourself if the gPTP is working as expected according to your use cases.

**Reply by aravind-gara** [reply]
@Jukkar Thanks a lot for quick response at discord and posting here the same.
Best regards,
araivnd
